README.md

```diff
@@ -35,4 +35,8 @@ You can use them 'as is' in the web API or import them in the python API as show
 - [`clean_mes_aides_situation.py`](./python/scripts/clean_mes_aides_situation.py) takes in a [Mes Aides](https://mes-aides.gouv.fr) situation and turns it into an OpenFisca-France situation.
 - [`generate_situation_example`](./python/scripts/generate_situation_examples) contains the `situation_example package`, along with the `try_situation_examples.py` script. The `situation_examples` package contains complete cookie cutter situations (single, couple, roommates) that are ready to use in the python API, and the web API (in their json form) 
 
-To run one of the following scripts, go to its directory an call it with the `python` command.
\ No newline at end of file
+To run one of the following scripts, go to its directory an call it with the `python` command.
+
+### [Frequently Asked Questions](./FAQ.md)
```

Comment:
Sounds misleading to me to have a big link written in English to a page in French. I think this title should be in French, or at least have `(in French)` in its title. Or link to a Google Translate link.